.travis.yml

```diff
@@ -26,6 +26,9 @@ env:
   - BUILD=site
   - BUILD=sonar
 
+matrix:
+  fast_finish: true
```

Comment:
Not sure, if this is exactly what we want:

https://docs.travis-ci.com/user/customizing-the-build/#sts=Fast%20Finishing
> If some rows in the build matrix are allowed to fail, ...

We don't have any build, that is allowed to fail. All our build jobs are required to determine the final build result...